<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2FLLKU8" failures="0" tests="1" name="Default test" time="0.61" errors="0" timestamp="30 Apr 2021 02:43:54 GMT">
  <testcase classname="com.surya.automation.fp.businessscripts.Gmail_Test" name="@BeforeTest initReports" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String)&amp;quot; because &amp;quot;com.surya.automation.base.Base_Test.extentTest&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String)" because "com.surya.automation.base.Base_Test.extentTest" is null
	at com.surya.automation.base.Base_Test.initReports(Base_Test.java:132)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest initReports -->
  <testcase classname="com.surya.automation.fp.businessscripts.Gmail_Test" name="@BeforeMethod beforeTCExecution" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod beforeTCExecution -->
  <testcase classname="com.surya.automation.fp.businessscripts.Gmail_Test" name="@AfterMethod afterTCExecution" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod afterTCExecution -->
  <testcase classname="com.surya.automation.fp.businessscripts.Gmail_Test" name="@AfterTest generateReports" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest generateReports -->
  <testcase classname="com.surya.automation.fp.businessscripts.Gmail_Test" name="gmail_Test" time="0.001">
    <skipped/>
  </testcase> <!-- gmail_Test -->
</testsuite> <!-- Default test -->
